PWGHF/HFC/Tasks/taskCorrelationDplusHadrons.cxx

```diff
@@ -297,22 +305,24 @@ struct HfTaskCorrelationDplusHadrons {
       float massD = candidate.mD();
       float ptD = candidate.ptD();
       float bdtScorePrompt = candidate.mlScorePrompt();
+      float bdtScoreNonPrompt = candidate.mlScoreNonPrompt();
       float bdtScoreBkg = candidate.mlScoreBkg();
       int effBinD = o2::analysis::findBin(binsPtEfficiencyD, ptD);
+      float bdtScorePromptorNonPrompt = isPromptAnalysis ? bdtScorePrompt : bdtScoreNonPrompt;
```

Comment:
```suggestion
      float bdtScorePromptOrNonPrompt = isPromptAnalysis ? bdtScorePrompt : bdtScoreNonPrompt;
```
Use camelCase properly. `Promptor` is not a word.

Reply:
I also modified it.